manim/_config/utils.py

```diff
@@ -313,7 +315,7 @@ class MyScene(Scene):
     }
 
     def __init__(self) -> None:
-        self._d = {k: None for k in self._OPTS}
+        self._d: dict[str, Any | None] = {k: None for k in self._OPTS}
```

Comment:
Is there a reason for `Any | None` over just `Any`?

Reply:
Altough `Any | None` is equivalent to `Any`, having the `None` explicitly stated is often used to let people know that it can indeed be `None` and they need to check for this. This is not really specified anywhere in docs or a PEP, but [used some times in typeshed](https://github.com/search?q=repo%3Apython%2Ftypeshed+%22Any+%7C+None%22&type=code)